### ISSUE TITLE
The payment MFE is still on Webpack 4.40

### ISSUE THREAD

**Issue by regisb** [openedx/build-test-release-wg#113]
On open-release/maple.master, the payment MFE still depends on Webpack 4.40, while other MFEs run on Webpack 5.50:

On open-release/maple.master: https://github.com/edx/frontend-app-payment/blob/eff6390a7fb30b99883a2fcb7fd2e0c029f680b1/package-lock.json#L1270
On master: https://github.com/edx/frontend-app-payment/blob/dbe08559f884cfc0324b72a3e89b4ae887d87f7f/package-lock.json#L1367

As a consequence, running the payment MFE in development is failing -- notably because of webpack-merge, which is used [here](https://github.com/overhangio/tutor-mfe/blob/b5f26c25e728f42d281f5f79e1714b1ac7884a4b/tutormfe/templates/mfe/apps/mfe/webpack.dev.config.js#L1), and for which the API has changed.

```
$ tutor dev start payment
payment_1                    |                                                                                                                                                                                     
payment_1                    | > @edx/frontend-app-payment@0.1.0 start /openedx/app                                                                                                                                
payment_1                    | > fedx-scripts webpack-dev-server --progress                                                                                                                                        
payment_1                    |                                                                                                                                                                                     
payment_1                    | Running with resolved config:                                                                                                                                                       
payment_1                    | /openedx/app/webpack.dev.config.js                                                                                                                                                  
payment_1                    |                                                                                                                                                                                     
payment_1                    | No local module configuration file found. This is fine.                                                                                                                             
payment_1                    | /openedx/app/webpack.dev.config.js:5                                                                                                                                                
payment_1                    | module.exports = merge(baseDevConfig, {                                                                                                                                             
payment_1                    |                  ^                                                                                                                                                                  
payment_1                    |                                                                                                                                                                                     
payment_1                    | TypeError: merge is not a function                                                                                                                                                  
payment_1                    |     at Object.<anonymous> (/openedx/app/webpack.dev.config.js:5:18)                                                                                                                 
payment_1                    |     at Module._compile (internal/modules/cjs/loader.js:999:30)                                                                                                                      
payment_1                    |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)                                                                                                       payment_1                    |     at Module.load (internal/modules/cjs/loader.js:863:32)                                                                                                                          payment_1                    |     at Function.Module._load (internal/modules/cjs/loader.js:708:14)                                                                                                                payment_1                    |     at Module.require (internal/modules/cjs/loader.js:887:19)                                                                                                                       payment_1                    |     at require (internal/modules/cjs/helpers.js:74:18)                                                                                                                              payment_1                    |     at WEBPACK_OPTIONS (/openedx/app/node_modules/webpack-cli/bin/utils/convert-argv.js:114:13)                                                                                     payment_1                    |     at requireConfig (/openedx/app/node_modules/webpack-cli/bin/utils/convert-argv.js:116:6)                                                                                        payment_1                    |     at /openedx/app/node_modules/webpack-cli/bin/utils/convert-argv.js:123:17                                                                                                       payment_1                    | npm ERR! code ELIFECYCLE                                                                                                                                                            payment_1                    | npm ERR! errno 1                                                                                                                                                                    payment_1                    | npm ERR! @edx/frontend-app-payment@0.1.0 start: `fedx-scripts webpack-dev-server --progress`                                                                                        payment_1                    | npm ERR! Exit status 1                                                                                                                                                              payment_1                    | npm ERR!                                                                                                                                                                            payment_1                    | npm ERR! Failed at the @edx/frontend-app-payment@0.1.0 start script.                                                                                                                payment_1                    | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.                                                                                  payment_1                    |                                                                                                                                                                                     payment_1                    | npm ERR! A complete log of this run can be found in:                                                                                                                                payment_1                    | npm ERR!     /root/.npm/_logs/2021-11-22T14_54_35_806Z-debug.log                                
```

**Comment by regisb** [comment]
It looks like this issue could be resolved simply by cherry-picking this commit: https://github.com/edx/frontend-app-payment/commit/dbe08559f884cfc0324b72a3e89b4ae887d87f7f (to be verified)

**Comment by cmltaWt0** [comment]
It looks like open-release/maple.master has [5.3.1 version](https://github.com/edx/frontend-app-payment/blob/open-release/maple.master/package-lock.json#L1234) and commit https://github.com/edx/frontend-app-payment/commit/dbe08559f884cfc0324b72a3e89b4ae887d87f7f has +5k -7k lines of changes and I'm not sure about safety to cherry-pick it.

**Comment by BbrSofiane** [comment]
I've opened the PR edx/frontend-app-payment#538

**Comment by BbrSofiane** [comment]
Should be fixed with edx/frontend-app-payment#541
